frontend_lib/src/helper.js

```diff
@@ -155,11 +155,17 @@ export const displayFileSize = (bytes, decimals) => {
   return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i]
 }
 
-// INFO - GB - 2019-09-03 - In this function each split is one separator that we change for a comma
-export const parserStringToList = string => {
-  let list = string.split(' ').join(',').split('\n').join(',').split(';').join(',').split(',')
-  list = list.filter(notEmptyString => notEmptyString !== '')
-  return list
+export const parserStringToList = (string, separatorList) => {
```

Comment:
as @lebouquetin proposed, you should give a default value to parameter separatorList.
That way, and since it'll always be the same value for separatorList, it'll make the function call simpler by not having to give the that parameter.